src/test/java/com/at/stepdefinitions/Get.java

```diff
@@ -1,51 +1,142 @@
+        String usersInfoString = api.retrieve(base.ServiceApi.hostName + base.apiResource).getBody();
+        JSONArray secondArray = new JSONArray(usersInfoString);
+
+        System.out.println("Users info in the DB: " + firstArray.toString());
+        System.out.println("Users info in the GET op: " + secondArray.toString());
+        QAUtils match = new QAUtils();
+        match.setNumberUsersDB(numberUsers);
+        boolean bool = match.compareDocumentsArrays(firstArray, secondArray);
+        if (bool) {
+            System.out.println("The values of all users match.");
+        } else {
+            System.out.println("The values doesn't match.");
+            throw new io.cucumber.java.PendingException();
+        }
     }
 
+    @Given("I want to retrieve 0 registered users") //ready
+    public void i_want_to_retrieve_0_registered_users() throws Exception {
+        base.apiResource = ApiPaths.SCE_GET_USERS;
+    }
+
+    @Then("the number of registered users should be 0") //ready
+    public void the_number_of_registered_users_should_be_0() {
+
+        QAUtils usersMongo = new QAUtils();
+        numberUsers = usersMongo.numberOfUsersDB(basedb.environment, basedb.uridb, "users");
+        System.out.println("The number of registered users in MongoDB is: " + numberUsers);
+
+        ApiTools api = new ApiTools(base.environment, base.apiResource);
+        String usersInfoString = api.retrieve(base.ServiceApi.hostName + base.apiResource).getBody();
+        JSONArray userGet = new JSONArray(usersInfoString);
+        System.out.println("The number of users retrieved by the GET operation is: " + userGet.length());
+        if (userGet.length() == numberUsers) {
+            System.out.println("The number of users match.");
+        }
+        //  throw new io.cucumber.java.PendingException();
+    }
+
+    @Then("information retrieved from service should be an empty list") //  ready
+    public void information_retrieved_from_service_should_be_an_empty_list() {
+        QAUtils Users = new QAUtils();
+        JSONArray firstArray = new JSONArray();
+        firstArray = Users.arrayUsersInfoDB(basedb.environment, basedb.uridb, "users");
+
+        ApiTools api = new ApiTools(base.environment, base.apiResource);
+        String usersInfoString = api.retrieve(base.ServiceApi.hostName + base.apiResource).getBody();
+        JSONArray secondArray = new JSONArray(usersInfoString);
+
+        System.out.println("Users info in the DB: " + firstArray.toString());
+        System.out.println("Users info in the GET op: " + secondArray.toString());
+        QAUtils match = new QAUtils();
+        match.setNumberUsersDB(numberUsers);
+        boolean bool = match.compareDocumentsArrays(firstArray, secondArray);
+        if (bool) {
+            System.out.println("The values of all users match.");
+        } else {
+            System.out.println("The values doesn't match.");
+            throw new io.cucumber.java.PendingException();
```

Comment:
Why throwing a pendingException?

Reply:
It was to see if the test fail, because above I implement a method that returns a boolean variable, if the boolean is true, print "The values of all users match", if boolean is false aplies  a throwing exception.
I didn't know if there is other way to show if the test fail.

Reply:
Done.
I used the Assert method.